Sources/MapboxNavigation/OrnamentsController.swift

```diff
@@ -58,6 +58,8 @@ extension NavigationMapView {
         
         var labelRoadNameCompletionHandler: (LabelRoadNameCompletionHandler)?
         
+        var eHorizonRoadName = [String]()
```

Comment:
Nit: I think that we mostly use `electronicHorizon`, so probably worth renaming to:
```suggestion
        var electronicHorizonRoadNames = [String]()
```

Reply:
Thanks for the suggestion, I'll improve this part.